clj-refactor.el

```diff
@@ -1058,10 +1058,15 @@ word test in it and whether the file lives under the test/ directory."
     (when (and cljr-add-ns-to-blank-clj-files
                (cljr--clojure-ish-filename-p (buffer-file-name))
                (= (point-min) (point-max)))
-      (clojure-insert-ns-form)
-      (newline 2)
-      (when (cljr--in-tests-p)
-        (cljr--add-test-declarations)))))
+      (let* ((ns (if (and (fboundp #'cider-expected-ns)
+                          (cider-connected-p))
+                     (cider-expected-ns)
+                   (clojure-expected-ns))))
+        (interactive)
+        (insert (format "(ns %s)" ns))
+        (newline 2)
```

Comment:
I know this line was already here before, but now is a good time to fix it. This `(newline 2)` can be removed if you change the format string above to `"(ns %s)\n\n"`.